test/models/message_test.dart

```diff
@@ -21,10 +23,18 @@ void main() {
       final message = MessageFactory().generateFake(
         read: false,
       );
+      whenRequest(messageProcessor, '');
+
       await message.markAsRead();
-      SupabaseManager.supabaseClient.from('messages').update({
```

Comment:
@istzen I stumbled over this when rewriting the mocking. This line should be unnecessary. There is no point in calling supabase, because HTTP is disabled and the only response will be our own mocks.

If you need to "change data" in supabase, you need to change the object and then renew the mock that returns the object on a GET request (see my RideDetail tests for example)

But in this case, that's not needed either. "message.read = true" is already set by the method itself. And we only need to know that the rpc function was called. You can see that verification below (note that it needed to be mocked in line 26 before the method call). We can't really test whether the supabase function actually marks the message as read, but that's not our job.

Reply:
yes thanks, i think that was the work of my friend and helper Copilot.